firebase-vscode/package.json

```diff
@@ -42,14 +42,6 @@
       {
         "command": "firebase.selectProject",
         "title": "Firebase: Switch project"
-      },
```

Comment:
I assumed there's no reason to keep those commands if the emulator is always started :)